test/fixtures/variant_keys.ftl

```diff
@@ -1,37 +1,37 @@
--simple-identifier =
-    {
+simple-identifier =
+    { $sel ->
```

Comment:
Can you cross-check which of these are already tested in `select_expressions.ftl `?

Reply:
I checked. `select_expressions.ftl` is mostly about selectors and things like empty variant values. In `variant_keys.ftl` I'd like to test different variant keys and their errors. I made a few changes in `select_expressions.ftl` to simplify the variant keys used there (`key` or `one` and `two` for nested placeables).